3_javascript/javascript1/week3/smartPhoneUsageApp/script.js

```diff
@@ -0,0 +1,48 @@
+let activities = [];
+// Adding an activity
+const addActivity = ( activity , duration) =>{
+    let today = new Date()
+    date = today.toLocaleDateString()
```

Comment:
Well done using Date here
if you want the date formate like (dd/mm/yyyy)
you can use 
`today.toLocaleDateString('en-GB')`